packages/trader/src/Modules/Page404/Components/Page404.jsx

```diff
@@ -6,6 +6,7 @@ import { localize } from '@deriv/translations';
 
 const Page404 = () => (
     <PageError
+        buttonOnClick={() => {}}
```

Comment:
Can we instead check if `buttonOnClick` prop is passed before invoking it in `PageError` component?

Reply:
Indeed, moved the check in the component. thanks for the pointers.